xml/System.IO/FileSystemWatcher.xml

```diff
@@ -643,7 +643,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Releases the unmanaged resources used by the <see cref="T:System.IO.FileSystemWatcher" />.</summary>
```

Comment:
I have no idea why, but the parameterless Dispose method does not appear in the build, while the Dispose(Boolean) method does. @dend and @joelmartinez, can you take a look?

Reply:
@dend @joelmartinez any chance you could help us? This is the only thing blocking us.

Reply:
Hmm ... this API doesn't seem to be added to any of the FrameworkIndex files (a file managed by mdoc). A visual inspection of the [`Dispose()` member](https://github.com/dotnet/dotnet-api-docs/blob/f121abe96f22ee4c71c4558bae03bfbee81debfc/xml/System.IO/FileSystemWatcher.xml#L619) suggests that it should be showing up. 

@mimisasouvanh I think this probably needs to be a devops bug to be looked at

@rpetrusha @carlossanlop it looks like this issue doesn't have anything to do with this particular PR, so it shouldn't hold up the merge. We'll have to look into the issue and resolve it in a future release of mdoc. Good find 👏

Reply:
Thank you!

Reply:
@joelmartinez @carlossanlop I know the PR was merged, but was there a devops bug created for the missing Dispose() member issue? If not, I can create it.

Reply:
I did not create an issue. Maybe @joelmartinez did?

Reply:
@mimisasouvanh I did not create a devops issue